src/displayapp/screens/settings/SettingSetTime.cpp

```diff
@@ -16,23 +18,31 @@ namespace {
   constexpr int16_t POS_Y_MINUS = 40;
   constexpr int16_t OFS_Y_COLON = -2;
 
-  void event_handler(lv_obj_t * obj, lv_event_t event) {
-    auto* screen = static_cast<SettingSetTime *>(obj->user_data);
+  void event_handler(lv_obj_t* obj, lv_event_t event) {
+    auto* screen = static_cast<SettingSetTime*>(obj->user_data);
     screen->HandleButtonPress(obj, event);
   }
 }
 
-SettingSetTime::SettingSetTime(Pinetime::Applications::DisplayApp *app, Pinetime::Controllers::DateTime &dateTimeController) :
-  Screen(app),
-  dateTimeController {dateTimeController} {
-  lv_obj_t * title = lv_label_create(lv_scr_act(), nullptr);
+struct Time12H {
+  std::string ampm;
```

Comment:
As only string literals are ever assigned to this member, wouldn't it be more memory efficient to use `const char *`?